docs/source/releases/v1_11_7a0.rst

```diff
@@ -23,6 +23,7 @@ Version 1.11.7a0 (2023-10-11)
 * Create "output_checker_kwargs" commandline argument
 * Convert base.py: plugin_module_to_obj to class-based method
 * Fixed from scipy.ndimage.interpolation import zoom deprecation warning
+* Raise error when two plugins have the same name
```

Comment:
NOTE: do not merge this until #238 has been merged.  When #238 has been merged to main, this one will automatically update so the destination is "main"